app/models/metric/ci_mixin.rb

```diff
@@ -1,5 +1,6 @@
 module Metric::CiMixin
   extend ActiveSupport::Concern
+  include SupportsFeatureMixin
```

Comment:
I think for modules you have to include it like https://github.com/isimluk/manageiq/blob/69901b69264cb03705514e126f8898955b129cc1/app/models/mixins/event_mixin.rb